src/mscorlib/src/System/ThrowHelper.cs

```diff
@@ -85,18 +85,39 @@ internal static class ThrowHelper {
                                                     Environment.GetResourceString(GetResourceName(resource)));
         }
 
+        internal static void ThrowArgumentOutOfRangeException(ExceptionArgument argument, int paramNumber, ExceptionResource resource)
+        {
+            throw new ArgumentOutOfRangeException(GetArgumentName(argument) + "[" + paramNumber + ']',
```

Comment:
Super nit picky: Looks like there is a misplaced space char before the `.ToString()` and there should be a space between the `ToString()` and `+`, e.g.:

Currently:
 
```c#
paramNumber .ToString()+ "]"
```

Should be:

```csharp
paramNumber.ToString() + "]"
```